airbyte-integrations/connectors/destination-gcs/src/main/resources/spec.json

```diff
@@ -229,6 +229,13 @@
                     }
                   }
                 ]
+              },
```

Comment:
Instead of having this appear multiple times in each format option, what if this was a field at the top level?

Reply:
Hi @davinchia. The is a Parquet option where this param is not applicable, so I had to add it several times for other applicable options (to avoid users confusion)

Reply:
I realised that after I typed this. This is fine.